packages/portal-proto/src/features/cohortBuilder/StickyControl.tsx

```diff
@@ -10,30 +10,45 @@ const StickyControl = ({
 }: {
   handleIsSticky: (isSticky: boolean) => void;
   isSticky: boolean;
-}): JSX.Element => (
-  <Tooltip
-    label={
-      isSticky ? "Unpin Cohort Bar" : "Pin Cohort Bar to top of Analysis Center"
-    }
-    withArrow
-    position="bottom"
-  >
-    <button
-      data-testid="button-cohort-bar-pin-unpin"
-      className="bg-primary-darker hover:bg-primary-darkest h-12 w-12 grid place-items-center text-white rounded-md"
-      onClick={() => handleIsSticky(!isSticky)}
-      aria-label="Toggle to pin or unpin cohort bar to top of Analysis Center"
+}): JSX.Element => {
+  const CheckboxIcon: CheckboxProps["icon"] = ({
+    indeterminate,
+    className,
+  }: {
+    indeterminate: boolean;
+    className: string;
+  }) =>
+    indeterminate ? (
+      <StickyOffIcon size="24px" className={className} />
+    ) : (
+      <StickyOnIcon size="24px" className={className} />
+    );
```

Comment:
Can this component be made as a separate one instead of a nested component within `StickyControl`?

Reply:
i dont see the point in splitting it off if its only going to be used in this one location, or are we following a pattern where we only have one component per file?